app/Http/Controllers/LeadContactController.php

```
@@ -1197,31 +1188,25 @@ public function patch(PatchRequest $request, $id)
-            Log::error('Lead patch update failed: ' . $e->getMessage(), [
+            Log::error('Lead patch update failed: '.$e->getMessage(), [
                 'lead_id' => $id,
                 'user_id' => user()->id,
-                'request_data' => $request->all()
+                'request_data' => $request->all(),
             ]);
 
-           
-            return Reply::error('An error occurred while updating the lead contact: ' . $e->getMessage());
-            
+            return Reply::error('An error occurred while updating the lead contact: '.$e->getMessage());
```

PHPStan (2.2.7)

[error] Method App\Http\Controllers\LeadContactController::patch() should return Illuminate\Http\Response but returns array.


(return.type)